compute/cloud-client/src/test/java/compute/custommachinetype/CustomMachineTypeIT.java

```diff
@@ -58,6 +60,8 @@ public class CustomMachineTypeIT {
 
   private ByteArrayOutputStream stdOut;
   private static InstancesClient instancesClient;
+  
+  @Rule public final MultipleAttemptsRule multipleAttemptsRule = new MultipleAttemptsRule(3);
```

Comment:
Tests still failed in Java11. IIC this is due to long wait for instance to be available for deletion. Can we make bigger timeouts or set them to grow exponentially in each rerun?

Reply:
Yes, added a 1.5 backoff timer with 5minutes between each retry. Thanks.